db/models.py

```diff
@@ -1,4 +1,7 @@
+from django.contrib.auth.models import AbstractUser
```

Comment:
Please do not delete the existing migration, create a new one if you need to.

Reply:
This comment is not implemented, return the first migration and make a second one with all required changes.